### PR TITLE
Add options attribute to gem_package resource

### PR DESCRIPTION
**Pull request by hico-horiuchi**
Add `options` attribute to `gem_package`.
(This attribute is same as [gem_package](https://docs.chef.io/resource_gem_package.html#properties) of Chef.)

For example:

```ruby
gem_package 'bundler' do
  options ['--no-ri', '--no-rdoc']
end
```

### CODE REVIEW

File: spec/integration/recipes/default.rb [hunk 1]
```diff
@@ -65,6 +65,10 @@
   version '1.2.2'
 end
 
+gem_package 'bundler' do
+  options ['--no-ri', '--no-rdoc']
```
Comment (ryotarai) [inline spec/integration/recipes/default.rb]: `--no-ri` and `--no-rdoc` are deprecated options. Please use `--no-document`

**Comment by hico-horiuchi** [comment]
@ryotarai Installed ruby at test is `1.9.1` , so `--no-document` options is not supported.
Should I back the options to `--no-ri --no-rdoc` ?
Or would you update ruby `2.0` and up?

```
DEBUG :       Executing `gem install --no-document bundler`...
DEBUG :         stdout | ERROR:  While executing gem ... (OptionParser::InvalidOption)
DEBUG :         stdout |     invalid option: --no-document
```

**Comment by ryotarai** [comment]
> Installed ruby at test is 1.9.1 , so --no-document option is not supported.
> Should I back the options to --no-ri --no-rdoc ?

For now, using `--no-ri --no-rdoc` in tests is better.

**Comment by hico-horiuchi** [comment]
I understand and remove the commit 8cc54c504d61a8a20857703f3eee83548da57f34 .

**Comment by ryotarai** [comment]
Thank you!

**Pull request merged**